### ISSUE TITLE
Question about image mean and std

### ISSUE THREAD

**Issue by shallowdream-x** [CoinCheung/BiSeNet#174]
Thank you for your code. I am learning from your code implementation. I am a beginner in deep learning. The question is about the mean and std in the demo.py. 
to_tensor = T.ToTensor(
    mean=(0.3257, 0.3690, 0.3223), # city, rgb
    std=(0.2112, 0.2148, 0.2115),
)
Where do these numbers come from, when I inference with my own image, I try to calculate the mean and std of my image to substitute it but I got bad result. So I want to know why. Thank you very much!

**Comment by MengzhangLI** [comment]
First, these mean and value `mean=(0.3257, 0.3690, 0.3223), std=(0.2112, 0.2148, 0.2115)` is related with author's cityscapes pretrained model, if your dataset or model is different from him, these settings may be harmful for your model.

Second, these values are used after normalization, i.e., it has been `div` 255 ([here](https://github.com/CoinCheung/BiSeNet/blob/44b230724fd031c22fe9ee12c2adbec3199e3656/lib/transform_cv2.py#L137)). So if you really want to use them, please check it out your image pixels whether undergoing normalization.

Best,

**Comment by CoinCheung** [comment]
@shallowdream-x 
Hi,
Much as @MengzhangLI mentioned, the mean/var values are computed from the training dataset. There are some tips that I think one need to know:
1. the input image should be first divided by 255 before normalized with mean/var values.
2. the mean/var values should be same during training/testing and inference stage, if you want the inference performance to be same as testing.
3. the mean/var values are not strictly required to be identical to those computed from training set. You can simple use `mean=(0.4, 0.4, 0.4), std=(0.2, 0.2, 0.2)`, which should bring limited difference.
